bin/PrintTokenGraph/README.md

```diff
@@ -0,0 +1,36 @@
+# Print token context graphs
+
+This utility helps to visualize PASTA's `TokenContext` feature, which allows one
+to connect tokens (`PrintedToken`) back to AST nodes (e.g. `Decl`, `Stmt`, etc.).
+This is a powerful feature, though it's not obvious how to use it absent the
+code in this example, hence why this example exists. Without seeing the context
+graphs produced by this example, it can also be hard to understand the value of
+the context graph printer.
+
+## Usage
+
+**Note:** Prior to using this tool, make sure the `/tmp/tokens` directory exists!
```

Comment:
Make configurable by a flag or is this some internal of the whole tool that must be satisfied?